Add FireNinja21

FireNinja21 opened the pull request:
My application belongs to Tier 1: I added myself to the Year Book. Fixed the profile picture issue from my second pull request. (Thank you for your patience & help!)

syedjahangirpeeran commented:
Hey @FireNinja21, I have checked your PR and found out that the uploaded image is of 314x315 pixels dimensions. Please upload an image with a minimum of 544x544 pixels. Also please insert the checklist in the description of the PR. Thanks.

FireNinja21 commented:
# Application to the Class of 2020🎓

This pull request template helps you complete an application to the **Class of 2020**. Use the checklist below to verify you have followed the instructions correctly. 

## Checklist ✅

- [x] I have read the instructions on the README file before submitting my application. 
- [x] I made my submission by creating a folder on the `_data` folder and followed the naming convention mentioned in the instructions (`<username>`), added my profile picture and markdown file.
- [x] I have used the Markdown file template to add my information to the Year Book.
- [x] I have completed the form: `https://airtable.com/shrmuHxu38ZariKJi`
- [x] My application belongs to:
  - [x] **Tier 1** 📖: I added myself to the Year Book.
  - [ ] ~~**Tier 2** ✏️: I wrote a post on DEV about a project I built.~~
  - [ ] ~~**Tier 3** ✨: I want my project to be highlighted on stream.~~
- [x] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [x] I understand I should not tag or add a reviewer to this Pull Request.
- [x] I understand the photo added to the template will be used in the ceremony "Graduate Walk". 
- [x] I have [added the event](http://www.google.com/calendar/event?action=TEMPLATE&dates=20200615T160000Z%2F20200615T183000Z&text=%24%20git%20remote%20%3Cgraduation%3E%20%F0%9F%8E%93&location=https%3A%2F%2Fwww.twitch.tv%2Fgithubeducation&details=) to my Calendar.

FireNinja21 commented:
# Application to the Class of 2020🎓

This pull request template helps you complete an application to the **Class of 2020**. Use the checklist below to verify you have followed the instructions correctly. 

## Checklist ✅

- [x] I have read the instructions on the README file before submitting my application. 
- [x] I made my submission by creating a folder on the `_data` folder and followed the naming convention mentioned in the instructions (`<username>`), added my profile picture and markdown file.
- [x] I have used the Markdown file template to add my information to the Year Book.
- [x] I have completed the form: `https://airtable.com/shrmuHxu38ZariKJi`
- [x] My application belongs to:
  - [x] **Tier 1** 📖: I added myself to the Year Book.
  - [ ] ~~**Tier 2** ✏️: I wrote a post on DEV about a project I built.~~
  - [ ] ~~**Tier 3** ✨: I want my project to be highlighted on stream.~~
- [x] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [x] I understand I should not tag or add a reviewer to this Pull Request.
- [x] I understand the photo added to the template will be used in the ceremony "Graduate Walk". 
- [x] I have [added the event](http://www.google.com/calendar/event?action=TEMPLATE&dates=20200615T160000Z%2F20200615T183000Z&text=%24%20git%20remote%20%3Cgraduation%3E%20%F0%9F%8E%93&location=https%3A%2F%2Fwww.twitch.tv%2Fgithubeducation&details=) to my Calendar.
